tinywallet/tinywallet/app.py

```diff
@@ -10,6 +10,10 @@
 from pathlib import Path
 import sys
 
+"""
+update your path to the decred folder accordingly
+"""
+sys.path.insert(0, '/home/silverback/PycharmProjects/tinydecred/decred')
```

Comment:
Can't have every user modify the source code to run properly. Need a better solution.

If user input is required, use environment variables.